PharmacyInfo/src/PageHeader.vue

```diff
@@ -0,0 +1,63 @@
+﻿<template>
+    <header>
+        <nav class="navbar navbar-expand-lg scrolling-navbar">
+            <a class="navbar-brand" href="../Medications.html">Pharmacy Info</a>
+            <div class="nav-item" v-if="role === 'Pharmacy'">
+                <a class="nav-link" href="../Patients.html">Patients</a>
+            </div>
+            <div class="nav-item">
+                <a class="nav-link" href="../Pharmacies.html">Pharmacies</a>
+            </div>
+            <div class="nav-item">
+                <a class="nav-link" href="../Medications.html">Medications</a>
+            </div>
+            <button v-on:click="logoutAndGoToAuthPage()">
```

Comment:
Please name methods that are used directly from UI according to the event on which they depend, in the current case it should be like 'onClickLogOut', and then you might call logoutAndGoToAuthPage() method inside onClickLogOut()